docs/src/api/class-request.md

```diff
@@ -185,7 +185,9 @@ Returns the matching [Response] object, or `null` if the response was not receiv
 
 ## method: Request.timing
 - returns: <[Object]>
-  - `startTime` <[float]> Request start time in milliseconds elapsed since January 1, 1970 00:00:00 UTC
+  - `issueTime` <[float]> Request issued time in milliseconds elapsed since January 1, 1970 00:00:00 UTC
```

Comment:
This structure corresponds to https://developer.mozilla.org/en-US/docs/Web/API/Resource_Timing_API/Using_the_Resource_Timing_API, we can leave it as is and 'fix' the startTime on Chromium / Firefox.

Reply:
If we are not going to add `issueTime` in here, How would we derive the `blocked` time? Do you want to expose the `requestSentTime` from `request` object?

Reply:
Would like some inputs here - was thinking like 
```
const blocked = timings.StartTime - timings.IssueTime
```

Reply:
Blocking time has a lax definition. We typically take the earliest probe: renderer request creation for subresource or browser request creation for navigation request. That's when the page decided it wants the request. After that it is queued and then some non-zero signal occurs in the network stack (present in resource timing). The delta between the first network stack number we get and the start is our queueing time.

Connection handshake (dns, ssl, etc) makes things harder because those can happen out of order per browser liking. So in the end, you will most likely want the delta between sending request and start.

Issue time does not really have any physical sense, I don't think it matters or even is defined. If there is a delta between request being sent and issue time, that's still blocking/queueing.

Reply:
Thanks a lot for the details on how the stack works. Keeping this in mind and also the RT API, I went through the dev-tools protocol and this is version I came up with. 

`request.IssueTime()` or `request.requestSentTime()` - Denotes the timestamp when the request was queued. 

Then we get the signal from the network stack via Resource Timing API - `timing.requestTime`(CDP), so to calculate blocking/queued we could do - https://github.com/microsoft/playwright/pull/7686/files#r671585936
```ts
const blocking = request.timing().startTime - request.issueTime() 
```

Also I have adjusted the timings to be compatible with RT spec - https://github.com/microsoft/playwright/pull/7686/commits/42a9fc997d78e1098d27c3591c6d97273fa2d48e#r671585936